src/tfvc/scm/resource.ts

```diff
@@ -37,10 +46,42 @@ export class Resource implements SCMResource {
      */
     public GetServerUri(): Uri {
         const serverItem: string = this._change.sourceItem ? this._change.sourceItem : this._change.serverItem;
-        const versionSpec: string = "C" + this._change.version;
+        // For conflicts set the version to "T"ip so that we will compare against the latest version
+        const versionSpec: string = this.HasStatus(Status.CONFLICT) ? "T" : "C" + this._change.version;
         return Uri.file(serverItem).with({ scheme: TfvcSCMProvider.scmScheme, query: versionSpec });
     }
 
+    public GetTitle(): string {
```

Comment:
This was moved here from the scmprovider class and then altered to handle conflicts.